core/operation/src/test/java/uk/gov/gchq/gaffer/operation/impl/LimitTest.java

```diff
@@ -45,7 +45,7 @@ public void builderShouldCreatePopulatedOperation() {
         assertThat(limit.getInput())
                 .hasSize(2);
         assertThat(limit.getResultLimit()).isEqualTo(1);
-        assertThat(limit.getInput()).containsOnly("1", "2");
+        assertThat((Iterable<String>) limit.getInput()).containsOnly("1", "2");
```

Comment:
Sorry I know this is a bit late in the process, but, I think instead of directly casting to `Iterable<String>`, you can let assertJ handle this itself by using the `.asList()` function.
Then this whole assertion could be cleaned up like so:
```java
assertThat(limit.getInput())
        .hasSize(2)
        .asList()
        .containsExactly("1", "2");
```

I know this is effectively doing the same thing but looks cleaner and allows for chaining.

Reply:
I copied the style/approach used by the other tests. The assertj approach does look nicer but I don't think we should have more than one style in each file, or even each set of tests. If we make piecemeal changes things will become inconsistent and confusing. A large separate PR to convert everything in `operation` (or other modules as well) to using assertj would be the way to go I think.

Reply:
That's probably fair, changing as we go is less effort at once but will lead to inconsistency.

Reply:
I'm with @t92549 on this. As you are editing the test files anyhow you could make the changes